pyiron/dft/waves/electronic.py

```diff
@@ -633,10 +670,10 @@ def plot_fermi_dirac(self):
             import matplotlib.pylab as plt
         except ModuleNotFoundError:
             import matplotlib.pyplot as plt
-        arg = np.argsort(self.eigenvalues)
-        plt.plot(
-            self.eigenvalues[arg], self.occupancies[arg], linewidth=2.0, color="blue"
-        )
+        for spin, eigenvalues in enumerate(self.eigenvalues):
+            arg = np.argsort(eigenvalues)
+            plt.plot(eigenvalues[arg], self.occupancies[spin][arg], linewidth=2.0, label="spin:{}".format(spin))
```

Comment:
only here I'd like to have something like `marker='-o'`

Reply:
done